collectors/ebpf.plugin/README.md

```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
 mount these filesystems on startup. More information can be found in
 the [ftrace documentation](https://www.kernel.org/doc/Documentation/trace/ftrace.txt).
 
+## Alerts
+
+Like other plugins, `eBPF.plugin` also have associated alerts for its charts. You can personalize them modifying
+the file `healt.d/ebpf.conf` using [`edit-config`](/docs/monitor/configure-alarms#edit-health-configuration-files).
+
+### Default alerts
+
+The following alerts are enabled by default:
+
+- retransmit_tcp_error: When at least 2% of packages are retransmitted, `netdata` raises alerts to notify possible
+  network problems.
+- apps_oomkill: When some apps group has an event of [Out Of Memory](#OOM-Killing), `netdata` sends a notification for
+  our users.
+
+### Silent alerts
+
+Certain Alerts are of high interest for specific use cases due to a safety policy or development purposes. To avoid
+alert overload, 'netdata' set them [silent](/guides/monitor/stop-notifications-alarms).
+
+-   task_error: This alert is raised every time a process/thread cannot be created.
+-   mount_call: For security reasons, some companies create policies that does not allow stick drivers to be used,
+    this alert sends notification every time a `mount (2)` or `umount (2)` `syscall` is called.
```

Comment:
> some companies create policies that do not allow stick drivers to be used

I think this one is very questionable.

Why do we need to check mount/umount calls? I think it is not a verbal agreement between users/administrators, but some technical constraint (inserting a USB stick doesn't result in `mount `call).

Also, creating/removing a docker container triggers many `mount`/`umount` calls.

Reply:
This is the motive we are setting it as silent. 
There are companies where users cannot extracted data from their environment, so they cannot mount/umount any stick drive their. I know companies that disable all USB when computers arrives in their building, and only after this users receive them.

Reply:
mount/umount call is not a warning event in general (default stock alarms), it is not even an info event. As I mentioned: you would get a warning active alarm after every `docker run ...`. I suggest removing it. I don't see how setting it to `silent` justifies it.

Reply:
@ilyam8 I removed completely the mount alarm and I also extend the time to test retransmitted packets.